bin/sgqlc-codegen

```diff
@@ -394,7 +395,7 @@ class %(name)s(sgqlc.types.Union):
 ''' % {
             'name': name,
             'schema_name': self.schema_name,
-            'types': ', '.join(v['name'] for v in t['possibleTypes']),
+            'types': ', '.join(types) if len(types) > 1 else types[0],
```

Comment:
I'll use a different approach, just add `,` to `__types__.

Reply:
Okey. Good)